homeassistant/components/litterrobot/vacuum.py

```diff
@@ -97,15 +103,43 @@ def status(self) -> str:
     async def async_turn_on(self, **kwargs: Any) -> None:
         """Turn the cleaner on, starting a clean cycle."""
         await self.robot.set_power_status(True)
+        if self._turn_on_call_reported:
+            return
+        self._turn_on_call_reported = True
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            "service_deprecation_turn_on",
+            breaks_in_ha_version="2024.2.0",
+            is_fixable=False,
```

Comment:
When creating repair issue for services we normally set `is_fixable` to `True` and `is_persistent` to `True`. That way the user can "fix" the issue after having updated their YAML.